adapters/repos/db/inverted/config_update.go

```diff
@@ -51,6 +56,18 @@ func validateBM25ConfigUpdate(initial, updated *models.InvertedIndexConfig) erro
 	return nil
 }
 
+func validateInvertedIndexConfigUpdate(initial, updated *models.InvertedIndexConfig) error {
```

Comment:
Method name seems a bit general. It is validating just particular parts of InvertedIndexConfig.
I would rename it to `validateIndexPropertyLengthAndNullStateUpdate` or split it to 2 separate methods `validateIndexPropertyLengthUpdate` and `validateIndexNullStateUpdate`

Reply:
Hmm, the method contains all validations we're doing for the Inverted Index so I think the name is fine. If anything else comes up in the future it can be added to this method.